### PR TITLE
Databases and schema

### PR DESCRIPTION
**Pull request by nicolasmelo1**
Adding Schema validation to palmares code, the idea is to abstract away stuff like Zod, Yup, valibot and others.

There is a genuine need for this: https://github.com/oven-sh/bun/issues/6797

### CODE REVIEW

File: .turbo/cookies/10.cookie
Comment (anti-duhring) [inline .turbo/cookies/10.cookie:1]: these empty files from `.turbo` are really necessary?

Comment (nicolasmelo1) [reply .turbo/cookies/10.cookie:1]: I think so, i did an update to turborepo and it automatically added those files. I'll take a look if it works when removing them

Comment (anti-duhring) [reply .turbo/cookies/10.cookie:1]: > I think so, i did an update to turborepo and it automatically added those files. I'll take a look if it works when removing them

in this case I'm gonna approve this PR and you decide about keeping these files or not

**Review by anti-duhring — APPROVED** [review]
looks good to me

**Pull request merged**